packages/now-cli/test/unit.js

```diff
@@ -56,7 +58,22 @@ const getStaticFiles = async (dir, isBuilds = false) => {
     strict: false,
   });
 
-  return getStaticFiles_(dir, nowConfig, { hasNowJson, output, isBuilds });
+  const files = await getStaticFiles_(dir, nowConfig, {
+    hasNowJson,
+    output,
+    isBuilds,
+  });
+  return normalizeWindowsPaths(files);
+};
+
+const normalizeWindowsPaths = files => {
+  if (process.platform === 'win32') {
+    const prefix = 'D:/a/now/now/packages/now-cli/test/fixtures/unit/';
+    return files.map(f =>
+      f.replace(new RegExp(/\\/g), '/').slice(prefix.length)
```

Comment:
Why `new RegExp()`? It's not even passing in a string 😛 

```suggestion
      f.replace(/\\/g, '/').slice(prefix.length)
```